pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -83,6 +84,32 @@ func findPlatformProvider(in *executor.Input) (name string, cfg *config.Platform
 	return
 }
 
+func LoadVpcConfiguration(in *executor.Input, vpcConfiguration string, ds *deploysource.DeploySource) (types.AwsVpcConfiguration, bool) {
+	in.LogPersister.Infof("Loading vpc manifest at commit %s", ds.Revision)
+
+	awsVpcConfiguration, err := provider.LoadVpcConfiguration(ds.AppDir, vpcConfiguration)
+	if err != nil {
+		in.LogPersister.Errorf("Failed to load VPC configuration (%v)", err)
```

Comment:
I'm not so concerned about it, but I want to unify whether `vpc` is lower or upper case.

Reply:
Since it's a proper noun, I'd like to capitalize it.